coalib/results/Diff.py

```diff
@@ -70,6 +72,57 @@ def from_string_arrays(cls, file_array_1, file_array_2, rename=False):
         return result
 
     @classmethod
+    def from_unified_diff(cls, unified_diff, original_file):
+        """
+        Creates a ``Diff`` object from given unified diff.
+
+        If this ``Diff`` is applied to the original file, the corrected
+        file will be created.
```

Comment:
What do you mean with this? Imo it's normal that you can apply a diff and that it changes file :)